packages/roosterjs-content-model/lib/editor/coreApi/createContentModel.ts

```diff
@@ -32,7 +32,7 @@ function internalCreateContentModel(
         ...(option || {}),
     };
 
-    if (core.reuseModel) {
+    if (core.reuseModel && option?.allowCacheElement !== false) {
```

Comment:
So that means default value is true?

In that case, suggest to add option as "disableCacheElement" which is more intuitive.

Reply:
default value is undefined. But in this scenario I set the value of the option to false. That way if the value is undefined or true we keep caching the elements.

Reply:
Understand. I mean, by default its value is undefined, but undefined has the same meaning with "true" here, which is not intuitive. In most cases people will think undefined == false. That is why I suggest to change the option name to be opposite one